### PR TITLE
Add Java 12, move to AdoptOpenJDK, and change default user to root

### PR DESCRIPTION
**Pull request by keeganwitt**
and re-order for better caching

### CODE REVIEW

**Comment by keeganwitt** [comment]
It looks like AdoptOpenJDK images are tagged with `aarch64` instead of `arm64v8`.  I'm reaching out on their Slack channel for clarification -- I'm wondering if they're using the tag they should be.

**Comment by keeganwitt** [comment]
PR pending changes to be done after #5710.

File: library/gradle [hunk 1]
```diff
@@ -34,10 +26,15 @@ Tags: 5.4.1-jdk11-slim, 5.4-jdk11-slim, jdk11-slim
 Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, ppc64le, s390x
 Directory: jdk11-slim
 
-Tags: 5.4.1-jre11, 5.4-jre11, jre11
-Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, ppc64le, s390x
-Directory: jre11
-
 Tags: 5.4.1-jre11-slim, 5.4-jre11-slim, jre11-slim
 Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, ppc64le, s390x
 Directory: jre11-slim
```
Comment (tianon) [inline library/gradle]: Looks like this entry should go away too? (no `jre11-slim` directory in https://github.com/keeganwitt/docker-gradle that I can see)

**Comment by keeganwitt** [comment]
Java 12 images are based on adoptopenjdk rather than openjdk, since there is no Java 12 image in that repo (not sure if we will or not).  If not, maybe those tags should have a different name to reflect that.

**Comment by tianon** [comment]
Appears to need a similar `Architectures:` trimming to https://github.com/docker-library/official-images/pull/5988#issuecomment-497463409; from Travis:

 - gradle:5.4.1-jdk11 (FROM openjdk:11-jdk) [arm32v5, arm32v7, i386, ppc64le, s390x]
 - gradle:5.4.1-jdk11-slim (FROM openjdk:11-jdk-slim) [arm32v5, arm32v7, i386, ppc64le, s390x]
 - gradle:5.4.1-jdk8 (FROM openjdk:8-jdk) [arm32v5, arm32v7, arm64v8, i386, ppc64le, s390x]
 - gradle:5.4.1-jdk8-slim (FROM openjdk:8-jdk-slim) [arm32v5, arm32v7, arm64v8, i386, ppc64le, s390x]
 - gradle:5.4.1-jre8 (FROM openjdk:8-jre) [arm32v5, arm32v7, arm64v8, i386, ppc64le, s390x]
 - gradle:5.4.1-jre8-slim (FROM openjdk:8-jre-slim) [arm32v5, arm32v7, arm64v8, i386, ppc64le, s390x]

> Would it be helpful for you if I took care of those removals, or would you prefer to do it?  Do you want to make a decision regarding naming of the `adoptopenjdk`-based tags before this merges?

**Comment by keeganwitt** [comment]
I did want to make a decision on adoptopenjdk tag first rather than push something out, then change it later.

**Comment by yosifkit** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-arches b/_bashbrew-arches
index 9f6a21b..11da032 100644
--- a/_bashbrew-arches
+++ b/_bashbrew-arches
@@ -1,68 +1,15 @@
-gradle:alpine @ amd64
-gradle:alpine @ arm32v6
-gradle:alpine @ arm64v8
-gradle:alpine @ i386
-gradle:alpine @ ppc64le
-gradle:alpine @ s390x
 gradle:jdk11 @ amd64
-gradle:jdk11 @ arm32v5
-gradle:jdk11 @ arm32v7
-gradle:jdk11 @ arm64v8
-gradle:jdk11 @ i386
 gradle:jdk11 @ ppc64le
 gradle:jdk11 @ s390x
-gradle:jdk11-slim @ amd64
-gradle:jdk11-slim @ arm32v5
-gradle:jdk11-slim @ arm32v7
-gradle:jdk11-slim @ arm64v8
-gradle:jdk11-slim @ i386
-gradle:jdk11-slim @ ppc64le
-gradle:jdk11-slim @ s390x
-gradle:jre11 @ amd64
-gradle:jre11 @ arm32v5
-gradle:jre11 @ arm32v7
-gradle:jre11 @ arm64v8
-gradle:jre11 @ i386
-gradle:jre11 @ ppc64le
-gradle:jre11 @ s390x
-gradle:jre11-slim @ amd64
-gradle:jre11-slim @ arm32v5
-gradle:jre11-slim @ arm32v7
-gradle:jre11-slim @ arm64v8
-gradle:jre11-slim @ i386
-gradle:jre11-slim @ ppc64le
-gradle:jre11-slim @ s390x
+gradle:jdk12 @ amd64
+gradle:jdk12 @ ppc64le
+gradle:jdk12 @ s390x
+gradle:jre12 @ amd64
+gradle:jre12 @ ppc64le
+gradle:jre12 @ s390x
 gradle:jre @ amd64
-gradle:jre @ arm32v5
-gradle:jre @ arm32v7
-gradle:jre @ arm64v8
-gradle:jre @ i386
 gradle:jre @ ppc64le
 gradle:jre @ s390x
-gradle:jre-alpine @ amd64
-gradle:jre-alpine @ arm32v6
-gradle:jre-alpine @ arm64v8
-gradle:jre-alpine @ i386
-gradle:jre-alpine @ ppc64le
-gradle:jre-alpine @ s390x
-gradle:jre-slim @ amd64
-gradle:jre-slim @ arm32v5
-gradle:jre-slim @ arm32v7
-gradle:jre-slim @ arm64v8
-gradle:jre-slim @ i386
-gradle:jre-slim @ ppc64le
-gradle:jre-slim @ s390x
 gradle:latest @ amd64
-gradle:latest @ arm32v5
-gradle:latest @ arm32v7
-gradle:latest @ arm64v8
-gradle:latest @ i386
 gradle:latest @ ppc64le
 gradle:latest @ s390x
-gradle:slim @ amd64
-gradle:slim @ arm32v5
-gradle:slim @ arm32v7
-gradle:slim @ arm64v8
-gradle:slim @ i386
-gradle:slim @ ppc64le
-gradle:slim @ s390x
diff --git a/_bashbrew-list b/_bashbrew-list
index 4e961b2..ce01537 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -1,57 +1,24 @@
 gradle:5.4
-gradle:5.4-alpine
 gradle:5.4-jdk
 gradle:5.4-jdk8
-gradle:5.4-jdk8-alpine
-gradle:5.4-jdk8-slim
 gradle:5.4-jdk11
-gradle:5.4-jdk11-slim
-gradle:5.4-jdk-alpine
-gradle:5.4-jdk-slim
+gradle:5.4-jdk12
 gradle:5.4-jre
 gradle:5.4-jre8
-gradle:5.4-jre8-alpine
-gradle:5.4-jre8-slim
-gradle:5.4-jre11
-gradle:5.4-jre11-slim
-gradle:5.4-jre-alpine
-gradle:5.4-jre-slim
-gradle:5.4-slim
+gradle:5.4-jre12
 gradle:5.4.1
-gradle:5.4.1-alpine
 gradle:5.4.1-jdk
 gradle:5.4.1-jdk8
-gradle:5.4.1-jdk8-alpine
-gradle:5.4.1-jdk8-slim
 gradle:5.4.1-jdk11
-gradle:5.4.1-jdk11-slim
-gradle:5.4.1-jdk-alpine
-gradle:5.4.1-jdk-slim
+gradle:5.4.1-jdk12
 gradle:5.4.1-jre
 gradle:5.4.1-jre8
-gradle:5.4.1-jre8-alpine
-gradle:5.4.1-jre8-slim
-gradle:5.4.1-jre11
-gradle:5.4.1-jre11-slim
-gradle:5.4.1-jre-alpine
-gradle:5.4.1-jre-slim
-gradle:5.4.1-slim
-gradle:alpine
+gradle:5.4.1-jre12
 gradle:jdk
 gradle:jdk8
-gradle:jdk8-alpine
-gradle:jdk8-slim
 gradle:jdk11
-gradle:jdk11-slim
-gradle:jdk-alpine
-gradle:jdk-slim
+gradle:jdk12
 gradle:jre
 gradle:jre8
-gradle:jre8-alpine
-gradle:jre8-slim
-gradle:jre11
-gradle:jre11-slim
-gradle:jre-alpine
-gradle:jre-slim
+gradle:jre12
 gradle:latest
-gradle:slim
diff --git a/gradle_alpine/Dockerfile b/gradle_alpine/Dockerfile
deleted file mode 100644
index 6d197d6..0000000
diff --git a/gradle_jdk11-slim/Dockerfile b/gradle_jdk11-slim/Dockerfile
deleted file mode 100644
index 7b59ce8..0000000
diff --git a/gradle_jdk11/Dockerfile b/gradle_jdk11/Dockerfile
index cb6758f..b356fba 100644
--- a/gradle_jdk11/Dockerfile
+++ b/gradle_jdk11/Dockerfile
@@ -1,10 +1,30 @@
-FROM openjdk:11-jdk
+FROM adoptopenjdk:11-jdk-hotspot
 
 CMD ["gradle"]
 
 ENV GRADLE_HOME /opt/gradle
-ENV GRADLE_VERSION 5.4.1
 
+RUN set -o errexit -o nounset \
+    && echo "Adding gradle user and group" \
+    && groupadd --system --gid 1000 gradle \
+    && useradd --system --gid gradle --uid 1000 --shell /bin/bash --create-home gradle \
+    && mkdir /home/gradle/.gradle \
+    && chown --recursive gradle:gradle /home/gradle \
+    \
+    && echo "Symlinking root Gradle cache to gradle Gradle cache" \
+    && ln -s /home/gradle/.gradle /root/.gradle
+
+VOLUME /home/gradle/.gradle
+
+WORKDIR /home/gradle
+
+RUN apt-get update \
+    && apt-get install --yes --no-install-recommends \
+       unzip \
+       wget \
+    && rm -rf /var/lib/apt/lists/*
+
+ENV GRADLE_VERSION 5.4.1
 ARG GRADLE_DOWNLOAD_SHA256=7bdbad1e4f54f13c8a78abc00c26d44dd8709d4aedb704d913fb1bb78ac025dc
 RUN set -o errexit -o nounset \
     && echo "Downloading Gradle" \
@@ -19,20 +39,5 @@ RUN set -o errexit -o nounset \
     && mv "gradle-${GRADLE_VERSION}" "${GRADLE_HOME}/" \
     && ln --symbolic "${GRADLE_HOME}/bin/gradle" /usr/bin/gradle \
     \
-    && echo "Adding gradle user and group" \
-    && groupadd --system --gid 1000 gradle \
-    && useradd --system --gid gradle --uid 1000 --shell /bin/bash --create-home gradle \
-    && mkdir /home/gradle/.gradle \
-    && chown --recursive gradle:gradle /home/gradle \
-    \
-    && echo "Symlinking root Gradle cache to gradle Gradle cache" \
-    && ln -s /home/gradle/.gradle /root/.gradle
-
-# Create Gradle volume
-USER gradle
-VOLUME "/home/gradle/.gradle"
-WORKDIR /home/gradle
-
-RUN set -o errexit -o nounset \
     && echo "Testing Gradle installation" \
     && gradle --version
diff --git a/gradle_jre11/Dockerfile b/gradle_jdk12/Dockerfile
similarity index 85%
copy from gradle_jre11/Dockerfile
copy to gradle_jdk12/Dockerfile
index 38a1c80..5c920c3 100644
--- a/gradle_jre11/Dockerfile
+++ b/gradle_jdk12/Dockerfile
@@ -1,10 +1,30 @@
-FROM openjdk:11-jre
+FROM adoptopenjdk:12-jdk-hotspot
 
 CMD ["gradle"]
 
 ENV GRADLE_HOME /opt/gradle
-ENV GRADLE_VERSION 5.4.1
 
+RUN set -o errexit -o nounset \
+    && echo "Adding gradle user and group" \
+    && groupadd --system --gid 1000 gradle \
+    && useradd --system --gid gradle --uid 1000 --shell /bin/bash --create-home gradle \
+    && mkdir /home/gradle/.gradle \
+    && chown --recursive gradle:gradle /home/gradle \
+    \
+    && echo "Symlinking root Gradle cache to gradle Gradle cache" \
+    && ln -s /home/gradle/.gradle /root/.gradle
+
+VOLUME /home/gradle/.gradle
+
+WORKDIR /home/gradle
+
+RUN apt-get update \
+    && apt-get install --yes --no-install-recommends \
+       unzip \
+       wget \
+    && rm -rf /var/lib/apt/lists/*
+
+ENV GRADLE_VERSION 5.4.1
 ARG GRADLE_DOWNLOAD_SHA256=7bdbad1e4f54f13c8a78abc00c26d44dd8709d4aedb704d913fb1bb78ac025dc
 RUN set -o errexit -o nounset \
     && echo "Downloading Gradle" \
@@ -19,20 +39,5 @@ RUN set -o errexit -o nounset \
     && mv "gradle-${GRADLE_VERSION}" "${GRADLE_HOME}/" \
     && ln --symbolic "${GRADLE_HOME}/bin/gradle" /usr/bin/gradle \
     \
-    && echo "Adding gradle user and group" \
-    && groupadd --system --gid 1000 gradle \
-    && useradd --system --gid gradle --uid 1000 --shell /bin/bash --create-home gradle \
-    && mkdir /home/gradle/.gradle \
-    && chown --recursive gradle:gradle /home/gradle \
-    \
-    && echo "Symlinking root Gradle cache to gradle Gradle cache" \
-    && ln -s /home/gradle/.gradle /root/.gradle
-
-# Create Gradle volume
-USER gradle
-VOLUME "/home/gradle/.gradle"
-WORKDIR /home/gradle
-
-RUN set -o errexit -o nounset \
     && echo "Testing Gradle installation" \
     && gradle --version
diff --git a/gradle_jre-alpine/Dockerfile b/gradle_jre-alpine/Dockerfile
deleted file mode 100644
index 117f780..0000000
diff --git a/gradle_jre-slim/Dockerfile b/gradle_jre-slim/Dockerfile
deleted file mode 100644
index 8ffa872..0000000
diff --git a/gradle_jre/Dockerfile b/gradle_jre/Dockerfile
index 260a652..d455138 100644
--- a/gradle_jre/Dockerfile
+++ b/gradle_jre/Dockerfile
@@ -1,10 +1,30 @@
-FROM openjdk:8-jre
+FROM adoptopenjdk:8-jre-hotspot
 
 CMD ["gradle"]
 
 ENV GRADLE_HOME /opt/gradle
-ENV GRADLE_VERSION 5.4.1
 
+RUN set -o errexit -o nounset \
+    && echo "Adding gradle user and group" \
+    && groupadd --system --gid 1000 gradle \
+    && useradd --system --gid gradle --uid 1000 --shell /bin/bash --create-home gradle \
+    && mkdir /home/gradle/.gradle \
+    && chown --recursive gradle:gradle /home/gradle \
+    \
+    && echo "Symlinking root Gradle cache to gradle Gradle cache" \
+    && ln -s /home/gradle/.gradle /root/.gradle
+
+VOLUME /home/gradle/.gradle
+
+WORKDIR /home/gradle
+
+RUN apt-get update \
+    && apt-get install --yes --no-install-recommends \
+       unzip \
+       wget \
+    && rm -rf /var/lib/apt/lists/*
+
+ENV GRADLE_VERSION 5.4.1
 ARG GRADLE_DOWNLOAD_SHA256=7bdbad1e4f54f13c8a78abc00c26d44dd8709d4aedb704d913fb1bb78ac025dc
 RUN set -o errexit -o nounset \
     && echo "Downloading Gradle" \
@@ -19,20 +39,5 @@ RUN set -o errexit -o nounset \
     && mv "gradle-${GRADLE_VERSION}" "${GRADLE_HOME}/" \
     && ln --symbolic "${GRADLE_HOME}/bin/gradle" /usr/bin/gradle \
     \
-    && echo "Adding gradle user and group" \
-    && groupadd --system --gid 1000 gradle \
-    && useradd --system --gid gradle --uid 1000 --shell /bin/bash --create-home gradle \
-    && mkdir /home/gradle/.gradle \
-    && chown --recursive gradle:gradle /home/gradle \
-    \
-    && echo "Symlinking root Gradle cache to gradle Gradle cache" \
-    && ln -s /home/gradle/.gradle /root/.gradle
-
-# Create Gradle volume
-USER gradle
-VOLUME "/home/gradle/.gradle"
-WORKDIR /home/gradle
-
-RUN set -o errexit -o nounset \
     && echo "Testing Gradle installation" \
     && gradle --version
diff --git a/gradle_jre11-slim/Dockerfile b/gradle_jre11-slim/Dockerfile
deleted file mode 100644
index d138554..0000000
diff --git a/gradle_jre11/Dockerfile b/gradle_jre12/Dockerfile
similarity index 85%
rename from gradle_jre11/Dockerfile
rename to gradle_jre12/Dockerfile
index 38a1c80..85490bb 100644
--- a/gradle_jre11/Dockerfile
+++ b/gradle_jre12/Dockerfile
@@ -1,10 +1,30 @@
-FROM openjdk:11-jre
+FROM adoptopenjdk:12-jre-hotspot
 
 CMD ["gradle"]
 
 ENV GRADLE_HOME /opt/gradle
-ENV GRADLE_VERSION 5.4.1
 
+RUN set -o errexit -o nounset \
+    && echo "Adding gradle user and group" \
+    && groupadd --system --gid 1000 gradle \
+    && useradd --system --gid gradle --uid 1000 --shell /bin/bash --create-home gradle \
+    && mkdir /home/gradle/.gradle \
+    && chown --recursive gradle:gradle /home/gradle \
+    \
+    && echo "Symlinking root Gradle cache to gradle Gradle cache" \
+    && ln -s /home/gradle/.gradle /root/.gradle
+
+VOLUME /home/gradle/.gradle
+
+WORKDIR /home/gradle
+
+RUN apt-get update \
+    && apt-get install --yes --no-install-recommends \
+       unzip \
+       wget \
+    && rm -rf /var/lib/apt/lists/*
+
+ENV GRADLE_VERSION 5.4.1
 ARG GRADLE_DOWNLOAD_SHA256=7bdbad1e4f54f13c8a78abc00c26d44dd8709d4aedb704d913fb1bb78ac025dc
 RUN set -o errexit -o nounset \
     && echo "Downloading Gradle" \
@@ -19,20 +39,5 @@ RUN set -o errexit -o nounset \
     && mv "gradle-${GRADLE_VERSION}" "${GRADLE_HOME}/" \
     && ln --symbolic "${GRADLE_HOME}/bin/gradle" /usr/bin/gradle \
     \
-    && echo "Adding gradle user and group" \
-    && groupadd --system --gid 1000 gradle \
-    && useradd --system --gid gradle --uid 1000 --shell /bin/bash --create-home gradle \
-    && mkdir /home/gradle/.gradle \
-    && chown --recursive gradle:gradle /home/gradle \
-    \
-    && echo "Symlinking root Gradle cache to gradle Gradle cache" \
-    && ln -s /home/gradle/.gradle /root/.gradle
-
-# Create Gradle volume
-USER gradle
-VOLUME "/home/gradle/.gradle"
-WORKDIR /home/gradle
-
-RUN set -o errexit -o nounset \
     && echo "Testing Gradle installation" \
     && gradle --version
diff --git a/gradle_latest/Dockerfile b/gradle_latest/Dockerfile
index 58f2ef2..cddee98 100644
--- a/gradle_latest/Dockerfile
+++ b/gradle_latest/Dockerfile
@@ -1,10 +1,30 @@
-FROM openjdk:8-jdk
+FROM adoptopenjdk:8-jdk-hotspot
 
 CMD ["gradle"]
 
 ENV GRADLE_HOME /opt/gradle
-ENV GRADLE_VERSION 5.4.1
 
+RUN set -o errexit -o nounset \
+    && echo "Adding gradle user and group" \
+    && groupadd --system --gid 1000 gradle \
+    && useradd --system --gid gradle --uid 1000 --shell /bin/bash --create-home gradle \
+    && mkdir /home/gradle/.gradle \
+    && chown --recursive gradle:gradle /home/gradle \
+    \
+    && echo "Symlinking root Gradle cache to gradle Gradle cache" \
+    && ln -s /home/gradle/.gradle /root/.gradle
+
+VOLUME /home/gradle/.gradle
+
+WORKDIR /home/gradle
+
+RUN apt-get update \
+    && apt-get install --yes --no-install-recommends \
+       unzip \
+       wget \
+    && rm -rf /var/lib/apt/lists/*
+
+ENV GRADLE_VERSION 5.4.1
 ARG GRADLE_DOWNLOAD_SHA256=7bdbad1e4f54f13c8a78abc00c26d44dd8709d4aedb704d913fb1bb78ac025dc
 RUN set -o errexit -o nounset \
     && echo "Downloading Gradle" \
@@ -19,20 +39,5 @@ RUN set -o errexit -o nounset \
     && mv "gradle-${GRADLE_VERSION}" "${GRADLE_HOME}/" \
     && ln --symbolic "${GRADLE_HOME}/bin/gradle" /usr/bin/gradle \
     \
-    && echo "Adding gradle user and group" \
-    && groupadd --system --gid 1000 gradle \
-    && useradd --system --gid gradle --uid 1000 --shell /bin/bash --create-home gradle \
-    && mkdir /home/gradle/.gradle \
-    && chown --recursive gradle:gradle /home/gradle \
-    \
-    && echo "Symlinking root Gradle cache to gradle Gradle cache" \
-    && ln -s /home/gradle/.gradle /root/.gradle
-
-# Create Gradle volume
-USER gradle
-VOLUME "/home/gradle/.gradle"
-WORKDIR /home/gradle
-
-RUN set -o errexit -o nounset \
     && echo "Testing Gradle installation" \
     && gradle --version
diff --git a/gradle_slim/Dockerfile b/gradle_slim/Dockerfile
deleted file mode 100644
index 69e1b0f..0000000
```

</details>

**Comment by yosifkit** [comment]
Build test of #5857; ec4e5e24; `amd64` (`gradle`):

```console

$ bashbrew build gradle:5.4.1-jdk8
Building bashbrew/cache:00ef5bb542c8d9fe71acf31feeffbab8ab61f6d90f01fcf5cb3aaab2fcf82e8b (gradle:5.4.1-jdk8)
Tagging gradle:5.4.1-jdk8
Tagging gradle:5.4-jdk8
Tagging gradle:jdk8
Tagging gradle:5.4.1-jdk
Tagging gradle:5.4-jdk
Tagging gradle:jdk
Tagging gradle:5.4.1
Tagging gradle:5.4
Tagging gradle:latest

$ test/run.sh gradle:5.4.1-jdk8
testing gradle:5.4.1-jdk8
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build gradle:5.4.1-jre8
Building bashbrew/cache:f29fadcee6485d168c32c7846ee3750cc9128e773c19647a7bf8984beee060a1 (gradle:5.4.1-jre8)
Tagging gradle:5.4.1-jre8
Tagging gradle:5.4-jre8
Tagging gradle:jre8
Tagging gradle:5.4.1-jre
Tagging gradle:5.4-jre
Tagging gradle:jre

$ test/run.sh gradle:5.4.1-jre8
testing gradle:5.4.1-jre8
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build gradle:5.4.1-jdk11
Building bashbrew/cache:56ffb01b8495b45f2019a2fe8d6e26602c3b6593ace8c34bb49663f8188f4b31 (gradle:5.4.1-jdk11)
Tagging gradle:5.4.1-jdk11
Tagging gradle:5.4-jdk11
Tagging gradle:jdk11

$ test/run.sh gradle:5.4.1-jdk11
testing gradle:5.4.1-jdk11
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build gradle:5.4.1-jdk12
Building bashbrew/cache:806bdec0af6a4b4b86eac1913a5ad681a70e15b2450d8401f13c8bcb9100cad1 (gradle:5.4.1-jdk12)
Tagging gradle:5.4.1-jdk12
Tagging gradle:5.4-jdk12
Tagging gradle:jdk12

$ test/run.sh gradle:5.4.1-jdk12
testing gradle:5.4.1-jdk12
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build gradle:5.4.1-jre12
Building bashbrew/cache:797692ff1b941324780e420853f3e4f4a95bd07c813601b49d786c5c44242c1f (gradle:5.4.1-jre12)
Tagging gradle:5.4.1-jre12
Tagging gradle:5.4-jre12
Tagging gradle:jre12

$ test/run.sh gradle:5.4.1-jre12
testing gradle:5.4.1-jre12
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed

```

**Pull request merged**